pysa_tutorial/exercise4/generate_models.py

```diff
@@ -26,19 +26,20 @@ class Ignore:
 def main() -> None:
     # Here, specify all the generators that you might want to call.
     generators = {
-        "get_REST_api_sources": generate_taint_models.RESTApiSourceGenerator(
```

Comment:
I like the idea of providing people a template of how to call the model generators, since this will be the first time in the tutorial that they've encountered it. The generators are not well documented right now, so it's also nice to keep this as a working example to point people to.

What if we leave `get_REST_api_sources` fully implemented here, and remove this line from the instructions for `operate_on_twos`:
```
1. Replace the TODO comment sections in `generate_models.py` with the correct dynamic model generator to taint a Django-style API
```

The instructions for `operate_on_threes` can be the first time the reader has to configure a model generator on their own, and they can copy-paste these lines as a starting point

Reply:
++, makes sense to me, thanks!